server/lib/queries.js

```diff
@@ -179,23 +179,20 @@ const getTopSponsors = () => {
 /**
  * Returns all the users of a group with their `totalDonations` and `role` (HOST/MEMBER/BACKER)
  */
-const getUsersFromGroupWithTotalDonations = (GroupIds) => {
+const getUsersFromGroupWithTotalDonations = (GroupIds, until) => {
+  const untilCondition = (table) => until ? `AND ${table}."createdAt" < '${until.toISOString().toString().substr(0,10)}'` : '';
+
   const groupids = (typeof GroupIds === 'number') ? [GroupIds] : GroupIds;
   return sequelize.query(`
-    WITH total_donations AS (
-      SELECT
-        max("UserId") as "UserId",
-        SUM(amount) as amount
-      FROM "Donations" d
-      WHERE d."GroupId" IN (:groupids) AND d.amount >= 0
-      GROUP BY "UserId"
-    ), last_donation AS (
+    WITH stats AS (
       SELECT
         max("UserId") as "UserId",
-        max("updatedAt") as "updatedAt"
+        SUM("amountInTxnCurrency") as "totalDonations",
```

Comment:
This will send the results in cents, instead of dollars. Are you dividing by a 100 somewhere else?